### DISCUSSION TITLE
Polygon isn't visible in ControlTheme

### DISCUSSION THREAD

**Discussion by Max-H-0** [AvaloniaUI/Avalonia#13653] [Q&A]
I can't get this polygon to show up while it's in the control theme:
![image](https://github.com/AvaloniaUI/Avalonia/assets/81490420/d1da4939-639e-46a2-ab40-7d93c770fd07)

It works exactly as expected if I put it directly in Design.PreviewWith:
![image](https://github.com/AvaloniaUI/Avalonia/assets/81490420/f54d74fe-6e90-4aeb-9133-25c8707abb1b)

What am I doing wrong here?

```xaml
<ResourceDictionary  xmlns="https://github.com/avaloniaui"
                     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                     xmlns:vm="clr-namespace:NodeEditor.ViewModels"
                     xmlns:c="clr-namespace:NodeEditor.Controls">
    <Design.PreviewWith>
        <c:Connector Width="10" Height="10"/>
    </Design.PreviewWith>

    <ControlTheme x:Key="{x:Type c:Connector}" TargetType="c:Connector">
        <Setter Property="Template">
            <ControlTemplate x:DataType="vm:ConnectorViewModel">
                <Polygon Width="10" 
                         Height="10" 
                         Fill="BlueViolet"
                         Stroke="Blue"
                         StrokeThickness="4"
                         Points="0,0 10,0 10,10 0,10"/>
            </ControlTemplate>
        </Setter>
    </ControlTheme>
</ResourceDictionary>
```

**Comment by timunie** [comment]
ResourceInclude for the control done correclty in the Demo-Project?

**Reply by Max-H-0** [reply]
I think so, other shapes like the ellipse work exactly as expected

**Reply by timunie** [reply]
Please attach a minimal sample to reproduce. Then I can analyze

**Reply by Max-H-0** [reply]
I think this is what you're asking for: https://github.com/Max-H-0/Test
I'm quite inexperienced with Avalonia, so I'd appreciate it if you'd also tell me if you notice any part of the project structure or naming that's kinda weird/wrong.

**Comment by robloo** [comment]
@Max-H-0 Which version of Avalonia are you using? I think this was a known bug https://github.com/AvaloniaUI/Avalonia/issues/11560 but it should be fixed in latest versions of Avalonia 11 (https://github.com/AvaloniaUI/Avalonia/pull/12651)

**Reply by Max-H-0** [reply]
I was using 11.0.2, I tried updating to 11.0.5 but it doesn't seem like it changed anything

**Reply by robloo** [reply]
Hm, @timunie we might want to re-open #11560. It seems to be the same issue and wasn't actually fixed or has regressed.

**Reply by timunie** [reply]
@robloo need to double check with latest nightly. If issue persists I will reopen or create a new one

**Reply by timunie** [reply]
in nightly it works for me at runtime. Designer shows nothing for me, but for no view actually. So not sure what is going on there. 
@Max-H-0 can you test on your side using latest nightly builds?

**Reply by timunie** [reply]
one more thing: https://github.com/Max-H-0/Test/blob/2f2787b32cbd7549572342dab345102c50048fe3/Test/App.axaml#L13

here you do a style include, but actually you have a resourcedict. So you need to do a ResourceInclude on App.Resources.

**Reply by Max-H-0** [reply]
Thank you for pointing that out.
I haven't checked github in a while, sorry for the late response.
What's nightly and how can I try that?

**Reply by timunie** [reply]
https://github.com/AvaloniaUI/Avalonia/wiki/Using-nightly-build-feed

**Reply by Max-H-0** [reply]
Nightly also works for me at runtime and I run into the same issue with the designer

**Reply by timunie** [reply]
Which IDE and which previewer version are you using? I guess if it works at runtime, this is the most important thing.

**Reply by Max-H-0** [reply]
I'm using Rider with AvaloniaRider 1.1.0
